services/gitdiff/csv.go

```diff
@@ -199,25 +181,82 @@ func createCsvDiff(diffFile *DiffFile, baseReader *csv.Reader, headReader *csv.R
 		}
 
 		for i := 0; i < len(a2b); i++ {
-			acell, _ := getCell(arow, i)
-			if a2b[i] == unmappedColumn {
-				cells[i] = &TableDiffCell{LeftCell: acell, Type: TableDiffCellDel}
+			var aCell string
+			aIsUndefined := false
```

Comment:
Probably better to set aIsUndefined := true and then only set it to false at line 193